st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -406,6 +406,11 @@ def post(self, spec_api, id, requester_user, no_merge=False, show_secrets=False)
         if list(set(spec_api.reset) - set(spec_api.tasks)):
             raise ValueError('List of tasks to reset does not match the tasks to rerun.')
 
+        delay = None
+
+        if hasattr(spec_api, "delay") and isinstance(spec_api.delay, int):
```

Comment:
I would just do ``delay = getattr(spec_api, 'delay', None)``, but that also works.